backend/src/yeomanui.ts

```diff
@@ -284,6 +320,10 @@ export class YeomanUI {
   }
 
   private exploreGenerators() {
+    const eventType = EVENT_TYPES.EXPLORE_AND_INSTALL_GENERATORS_LINK;
+    getSWA().track(eventType);
+    this.logger.trace("SAP Web Analytics tracker was called", {eventType});
+
```

Comment:
wrap this into a func. can be placed at /swa-tracker/swa-tracker-wrapper.ts or at /swa-tracker/swa-tracker-utils.ts

Reply:
Yes, I had doubts whether to create a function here as well.